Tools/generate-docc-reference/GenerateDoccReference.swift

```diff
@@ -126,4 +153,19 @@ struct GenerateDoccReference: ParsableCommand {
       print(page)
     }
   }
+
+  func assignSectionTitle(to argument: ArgumentInfoV0) throws -> String {
```

Comment:
I don't see any errors that this function is throwing, which would eliminate the need for the new `argumentAssignmentFailed` error above.